pkgs/applications/file-managers/nnn/default.nix

```diff
@@ -34,6 +34,7 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   patches = [
+    ./darwin-fix-file-mime-opts.patch
```

Comment:
Could you please add a comment explaining why this patch is needed and when it should be removed (eg. Is there an issue upstream about it?)

Also: https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/missing-patch-comment.md

Reply:
I added some comments according to the guideline. This issue is Nix-specific, so there is no PR on upstream. The Nix file command has different options from the macOS-default one, which is why nnn is not working correctly.